embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/worker/WorkerThreadModule.kt

```diff
@@ -31,15 +31,10 @@ internal interface WorkerThreadModule : Closeable {
  */
 internal enum class ExecutorName(internal val threadName: String) {
     BACKGROUND_REGISTRATION("background-reg"),
-    SCHEDULED_REGISTRATION("scheduled-reg"),
     CACHED_SESSIONS("cached-sessions"),
     SEND_SESSIONS("send-sessions"),
```

Comment:
Is this the thread that preps the session payload before sending it out to the `DeliveryService` to send?

Reply:
This is probably next on my target list - it's actually used to load background activities + moments & initiate HTTP requests via the API service. I'm not convinced it is necessary